browser/css/notebookbar.css

```diff
@@ -453,27 +453,15 @@
 	height: 60px;
 	width: 530px;
 	overflow-x: hidden;
-	padding: 5px;
-	padding: 0px;
+	padding: 2px;
 	border: 1px solid var(--gray-color);
+	border-radius: var(--border-radius);
 	border-collapse: separate; /*To use box-shadow in Internet Explorer 9 or later*/
-	-webkit-box-shadow: 0 0 3px var(--gray-color);
-	box-shadow: inset 0 0 3px var(--gray-color);
```

Comment:
This seems unrelated to the commit and related to how the main parent looks. Could you please cut this from the current commit and add another one just with that? Let me know if you need help with that

Reply:
sure will make a separate PR with how select, hover, ... should look like.

Reply:
What I said it's still true: This chunk of changes seems unrelated to the commit and related to how the main parent element looks. Could you please:
a) removing that from the current commit and add another commit just with that? 
b) Or if it's difficult for you to do that, no worries, just then please add amend the commit and add in the description that this commit also changes the parent elements' look n feel (`#stylesview`)